src/cf.data.pre

```diff
@@ -6445,6 +6445,7 @@ DOC_START
 		 reload-into-ims
 		 ignore-reload
 		 ignore-no-store
+		 ignore-must-revalidate
```

Comment:
What happens to folks that had ignore-must-revalidate in their old Squid configurations when they update to this new Squid code _without_ going through any Squid version that does not have ignore-must-revalidate? Will there be any changes in relevant Squid behavior (while their configuration remains unchanged)? If yes, we should disclose those changes, either in the PR description or Squid release notes. Personally, I prefer the latter.